src/controllers/auth.js

```diff
@@ -1,11 +1,61 @@
+const jwt = require('jsonwebtoken');
```

Comment:
Once you merge `main` branch you can accept all changes made by Perizat inside this file.